orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteStageHandler.kt

```diff
@@ -87,7 +89,10 @@ class CompleteStageHandler(
           stage.status = status
           stage.endTime = clock.millis()
         } catch (e: Exception) {
-          log.error("Failed to construct after stages for $stage.id", e)
+          log.error("Failed to construct after stages for ${stage.name} ${stage.id}", e)
```

Comment:
`log.error("Failed blah blah for stage ${stage.name}", e, kv("stageId", stage.id))` is pretty neat-o too.

Reply:
cool, thanks!